addons/medical_treatment/functions/fnc_treatment.sqf

```diff
@@ -54,88 +54,90 @@ private _userAndItem = if (GET_NUMBER_ENTRY(_config >> "consumeItem") == 1) then
 
 _userAndItem params ["_itemUser", "_usedItem"];
 
-// Get treatment animation for the medic
-private _medicAnim = if (_medic isEqualTo _patient) then {
-    getText (_config >> ["animationMedicSelf", "animationMedicSelfProne"] select (stance _medic == "PRONE"));
-} else {
-    getText (_config >> ["animationMedic", "animationMedicProne"] select (stance _medic == "PRONE"));
-};
+if !(_medic isEqualTo player && {!isNull findDisplay 312}) then {
```

Comment:
```suggestion
if (_medic isNotEqualTo player && {!isNull findDisplay 312}) then {
```
If you meant "the medic ain't the Player and the current view is the Zeus interface", if I understand [`findDisplay`](https://community.bistudio.com/wiki/findDisplay) correctly.

Reply:
I want to show the animation unless:
1. the medic is the player
2. the player is in Zeus

Your change would mean that the animation would never be shown for the player. An alternative could be this, which I suppose could be easier to read.
```suggestion
if (_medic isNotEqualTo player || {isNull findDisplay 312}) then {
```

Reply:
Yeah, that's much more readable.